installation/nuget.md

```diff
@@ -20,14 +20,29 @@ You can set up the remote Telerik NuGet feed in the following ways:
 * [Use the .NET CLI](#use-the-net-cli)
 * [Edit the Nuget.Config file](#edit-the-nuget-config-file)
 
->tip When working with the .NET CLI or editing the `NuGet.Config` manually, you can use your Telerik account credentials or a [NuGet API Key](#use-nuget-api-key). If you are logging in to telerik.com through single sign-on (SSO), use a [NuGet API Key](#use-nuget-api-key).
+Regardless of how you set up the Telerik NuGet feed, you must first [generate a NuGet API key](#generate-a-nuget-api-key) needed for the authentication.
 
->warning Never hard-code Telerik account credentials or NuGet API keys in a `NuGet.Config` file in a GitHub repository, Docker image, or any location that may be accessed by unauthorized parties. A NuGet key is valuable and bad actors can use it to access the NuGet packages that are licensed under your account. A credentials abuse can lead to a review of the affected Telerik account.
+>warning Never hard-code your Telerik NuGet API keys in a `NuGet.Config` file in a GitHub repository, Docker image, or any location that may be accessed by unauthorized parties. A NuGet key is valuable and bad actors can use it to access the NuGet packages that are licensed under your account. A credentials abuse can lead to a review of the affected Telerik account.
 
 For NuGet-related issues, see [NuGet Feed Troubleshooting](slug:troubleshooting-nuget).
 
 For information on automated builds, CI and CD, see [CI, CD, Build Server](slug:deployment-ci-cd-build-pc).
 
+## Generate a NuGet API Key
+
+As the Telerik NuGet server requires authentication, the first step is to obtain an API key that you will use instead of a password. Using an API key instead of a password is a more secure approach, especially when working with [.NET CLI](#use-the-net-cli) or the [`NuGet.Config` file](#edit-the-nugetconfig-file).
+
+1. Go to the [NuGet Keys](https://www.telerik.com/account/downloads/nuget-keys) page in your Telerik account.
+1. Click **Generate New Key +**.
+
+   ![Manage NuGet Keys](../installation/images/account-generate-nuget-api-key.png)
```

Comment:
We do not use screenshots, because they are harder to maintain and usually not necessary. There are two exceptions - animated screenshots and screenshots with annotations.